File: python-client-sdk/src/orbapi/models/shared/webhookmetadatacreate.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from orbapi import utils


@dataclass_json
@dataclasses.dataclass
class WebhookMetadataCreate:
    r"""WebhookMetadataCreate
    Information describing a webhook
    """
    
    webhook_url: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('webhookUrl') }})
    

File: typescript-client-sdk/src/sdk/models/shared/webhookmetadatacreate.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// WebhookMetadataCreate
/** 
 * Information describing a webhook
**/
export class WebhookMetadataCreate extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=webhookUrl" })
  webhookUrl?: string;
}
